saleor/webhook/event_types.py

```diff
@@ -190,6 +195,7 @@ class WebhookEventAsyncType:
         PAGE_DELETED: PagePermissions.MANAGE_PAGES,
         TRANSLATION_CREATED: SitePermissions.MANAGE_TRANSLATIONS,
         TRANSLATION_UPDATED: SitePermissions.MANAGE_TRANSLATIONS,
+        OBSERVABILITY: AppPermission.MANAGE_APPS,
```

Comment:
Are you sure of this? Don't you think that better way, would be having separate permission for observability? I am not conviced that app that, gathers a whole communication logs should use `manage_apps`

Reply:
Ok, working on that